src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -74,4 +75,13 @@ describe('Test suite for dictionary concepts components', () => {
     const inputs = wrapper.find('select');
     expect(inputs).toHaveLength(2);
   });
+  it('should render the new source to equal MapType', () => {
```

Comment:
How does this test ensure that a user can edit the 'relationship' field appropriately?

Reply:
I have refactored the test.  Is it clearer now? Otherwise, please elaborate on what I'm missing.